packages/calcite-components/src/utils/focusTrapComponent.ts

```diff
@@ -51,20 +51,24 @@ export function connectFocusTrap(component: FocusTrapComponent, options?: Connec
   component.focusTrap = createFocusTrap(focusTrapNode, createFocusTrapOptions(el, options?.focusTrapOptions));
 }
 
+const outsideClickSet = new WeakSet<HTMLElement | SVGElement>();
```

Comment:
Can you add a brief explainer or rename this to clarify its intent? If I'm following correctly, this is used to determine the deactivate/close cause during a component's focus return, right?

Reply:
This is used to store a component that where the trap is being deactivated due to an outside click. Any suggestions on the name of the constant?

- deactivatedViaOutsideClickSet?
- outsideClickDeactivated?

Reply:
`outsideClickDeactivated` gets my vote. ☑️